pull_request_template.md

```diff
@@ -9,3 +9,51 @@
 ## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.
 
 ## Does this PR require any French translations? Have they been included?
+
+## QA checklist: complete all parts relevant to your ticket changes
+
+### Explore
+
+- Check Desktop and on Mobile
```

Comment:
Could we try to figure out how to make these show up as checkboxes?

Reply:
Yes good idea